scipy/stats/tests/test_distributions.py

```diff
@@ -1462,6 +1462,20 @@ def test_fit_MLE_comp_optimzer(self, rvs_shape, rvs_loc, rvs_scale):
         _assert_less_or_close_loglike(stats.pareto, data, func, floc=0,
                                       fscale=rvs_scale/2)
 
+        # previously, the overridden fit method required `floc` to be fixed
+        # in order to find a fit. Those assertions are above. However,
+        # the addition of solving a system of partial derivatives makes it
+        # possible to fit data when `floc` is free.
+        _assert_less_or_close_loglike(stats.pareto, data, func)
+
+        # When `floc` is free, the fit method optimizes over the scale with a
+        # system of partial derivatives. It is possible to fix the shape in
+        # this system, even to a non-optimal value and still get a better fit.
+        _assert_less_or_close_loglike(stats.pareto, data, func, f0=rvs_shape)
+        _assert_less_or_close_loglike(stats.pareto, data, func, f0=rvs_shape+1)
+
+        # seed that produces issues: 176979
```

Comment:
I don't remember which combination of parameters this seed was broken for. 

This is the script to find the bad seeds. https://gist.github.com/2255009e6668fb7174435afa3a13f48f